infra/experimental/sanitizers/ExecSan/PoEs/python-libnmap-0.7.2/libnmap_target.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Invoking the target with the inputs from fuzzers as its parameters."""
+
+import sys
```

Comment:
Can you add a link to the advisory details too?

Reply:
+1